src/commands.cpp

```diff
@@ -8335,4 +8335,115 @@ bool CommandDismissBanner::procresult(Result r)
     return r.wasErrorOrOK();
 }
 
+bool CommandFetchGoogleAds::procresult(Command::Result r)
+{
+    string_map result;
+    if (r.wasErrorOrOK())
+    {
+        mCompletion(r.errorOrOK(), result);
```

Comment:
Is it possible that the command returns no ads, so it simply returns an `API_OK`?? <-- `[0]`
If so, then this code is fine. If not possible, then we may want to check for strictly error and return an API_INTERNAL if not error or object with ads.